templates/results.html

```diff
@@ -15,17 +15,112 @@ <h5 style="color: white;">{{ e.name }} to {{  e.target["name"] }} </h5>
+  		<a class="btn btn-info btn-email mb-1" data-toggle="modal" data-target="#newsletterModal" onclick="copy_compose('yahoo', {{loop.index}}, '{{e.target['email']}}', '{{e.mailto_subject}}')"role="button">Open in Yahoo Mail</a>
 
   		<br>
   	{% endfor %}
   </div>
+
+  <div class="modal fade" id="newsletterModal" tabindex="-1" role="dialog" aria-labelledby="newsletterModalLabel" aria-hidden="true">
+	<div class="modal-dialog" role="document">
+	  <div class="modal-content">
+		<div class="modal-header">
+		  <h5 class="modal-title" id="newsletterModalLabel">Newsletter</h5>
+		  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+			<span aria-hidden="true">&times;</span>
+		  </button>
+		</div>
+		<div class="modal-body">
+			<div>
+				Consider subscribing to our newsletter to receive updates about upcoming features such as Template Maker!
+			</div>
+		  <form role="form" id="newsletter-form">
+			  <div class="form-group">
+				  <label class="control-label">E-Mail Address</label>
+				  <div>
+					<input type="email" class="form-control" name="email" placeholder="example@domain.com">
+				  </div>
+			  </div>
+
+			  <div class="btn_toolbar">
+				<input class="btn btn-success mb-1" type="submit" value="Subscribe">
+				<button class="btn btn-danger mb-1" data-dismiss="modal">Dismiss</button>
+			</div>
+		  </form>
+
+
+		</div>
+	  </div>
+	</div>
+  </div>
+
+
+  <div class="modal fade" id="resultModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+	<div class="modal-dialog" role="document">
+	  <div class="modal-content">
+		<div class="modal-header">
+		  <h5 class="modal-title" id="exampleModalLabel">Newsletter</h5>
+		  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+			<span aria-hidden="true">&times;</span>
+		  </button>
+		</div>
+		<div class="modal-body">
+			<div id="result">
+			</div>
+		</div>
+		<div class="modal-footer">
+			<button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+		</div>
+	  </div>
+	</div>
+  </div>
+
   <script type="text/javascript">
 
+  function subSubmit(ev)
+  {
+	  ev.preventDefault();
+	  var request = new XMLHttpRequest();
+	  request.addEventListener('load', subShow);
+	  request.open('POST', {{ url_for('subscribe_to_newsletter')|tojson }});
+	  request.send(new FormData(this));
+	  $('#newsletterModal').modal('hide');
+  }
+
+  function subShow()
+  {
+	  var data = JSON.parse(this.responseText);
+	  if (data.status == 'success')
+	  {
+		  var div = document.getElementById("result");
+		  var content = document.createTextNode("You have succesfully subscribed to our newsletter, Thank you!");
+		  div.appendChild(content)
+		  $('#resultModal').modal('show');
```

Comment:
I don't usually like JS but this is pretty, nice one!

Reply:
Thanks!